src/js/demo.js

```diff
@@ -14,8 +14,8 @@ new Vue({
 		rules: {
 			url: 'url',
 			dateISO: 'dateISO',
-			integer: 'integer',
-			numeric: 'numeric',
+			integer: 'integer,min,3,max,10',
```

Comment:
I would like it to be something like ```integer,min:3, max:10``` and the validation rule to be applied just when this pattern is present.